xcm/xcm-builder/src/nonfungibles_adapter.rs

```diff
@@ -42,7 +42,7 @@ impl<
 		context: &XcmContext,
 	) -> result::Result<xcm_executor::Assets, XcmError> {
 		log::trace!(
-			target: "xcm::non_fungibles_adapter",
+			target: "xcm::nonfungibles_adapter",
```

Comment:
Please create a constant `LOG_TARGET: &str = "xcm::nonfungibles_adapter"` instead of copying thus here all the time.